script/Arcueid/Day 14/14_00_ARC14_1.txt

```diff
@@ -1271,5 +1273,5 @@ For as long as I can remember―――
 [sha:9920780df71b832b6f4071c546deb29a826da68e]{
 -- Page 78, Offset 37002.
 -- 夜が明けるまで、いつまでも見上げていた。
-I look up at the moon until the day dawns.
+I look up at the moon until the day breaks.
```

Comment:
Then glory be to god.

Reply:
![image](https://user-images.githubusercontent.com/97076550/158506101-e10f6b28-9743-4f28-9407-b133257ec023.png)